conf/scripts/route-down.d/40-vpnclient-unset-ipv6

```diff
@@ -2,10 +2,12 @@
 
 is_ip6addr_set() {
   local ip6_addr=${1}
-  ip address show dev tun0 2> /dev/null | grep -q "${ip6_addr}/"
+  ip address show dev "${gateway_interface}" 2> /dev/null | grep -q "${ip6_addr}/"
 }
 
 ip6_addr=$(yunohost app setting "vpnclient" "ip6_addr")
+gateway_interface=${dev}
```

Comment:
Eeeh not sure to understand where `$dev` is coming from ? Is this coupled to the way openvpn invoke the script ?

Reply:
```suggestion
# cf https://openvpn.net/community-resources/reference-manual-for-openvpn-2-4/#environmental-variables for where 'dev' comes from
gateway_interface=${dev}
```